api/events.proto

```diff
@@ -57,6 +57,8 @@ message Event {
   string remote_ip = 10 [(gogoproto.customname) = "RemoteIP"];
   // The IP address of the caller that triggered this event.
   string user_agent = 11;
+  // The unique identifier of the event, assigned on creation.
+  string unique_id = 12 [(gogoproto.customname) = "UniqueID"];
```

Comment:
After testing this, it appears like this prop is not forwarded to the event object that is received from the HTTP stream.

Reply:
Are you sure you've merged this locally correctly ? I can see it in my browser request logs:

```
{"result":{"name":"events.stream.start","time":"2020-09-10T13:40:45.527547125Z","identifiers":[{"application_ids":{"application_id":"app1"}}],"correlation_ids":["rpc:/ttn.lorawan.v3.Events/Stream:01EHW3YGAB9XQC95F4DXQYSMRB"],"origin":"A98BCD2222F1","unique_id":"01EHW3YGAQW0C2S226MW9ENH1B"}}

{"result":{"name":"end_device.update","time":"2020-09-10T13:41:19.508426107Z","identifiers":[{"device_ids":{"device_id":"dev1","application_ids":{"application_id":"app1"}}}],"data":{"@type":"type.googleapis.com/google.protobuf.Value","value":["name"]},"correlation_ids":["rpc:/ttn.lorawan.v3.EndDeviceRegistry/Update:01EHW3ZHFVJ3T0FXW78YMT8M7R"],"origin":"A98BCD2222F1","visibility":{"rights":["RIGHT_APPLICATION_DEVICES_READ"]},"unique_id":"01EHW3ZHGM15PCT4S5PVZBTJJB"}}
```

Reply:
Indeed, I had my console stack config still set to the staging server :facepalm: . Works now!